api/types/client.go

```diff
@@ -252,4 +253,9 @@ type VanClientInterface interface {
 	GetVersion(component string, name string) string
 	GetIngressDefault() string
 	RevokeAccess(ctx context.Context) error
+	CreateServiceCA(ownerRef *metav1.OwnerReference) error
```

Comment:
As these are resource operations, suggest not adding to the client interface/model and look at leveraging/enhancing the associated pkg routines.